staging/cse/windows/README

```diff
@@ -1,11 +1,5 @@
 # Unit Tests
-1. Please install [pester](https://github.com/pester/Pester) with the command `Install-Module -Name pester -SkipPublisherCheck -Force`.
-  - Doc: https://pester.dev/docs/quick-start
-  - Courses: https://docs.microsoft.com/en-us/shows/testing-powershell-with-pester/
-1. You need to run unit tests on Windows. NOTE: WSL does not work.
-1. Run unit tests
-  - `cd staging\cse\windows\`
-  - `.\cse.tests.ps1`
+Unit Tests are executed on pipeline. The yaml file of unit test pipeline is .github/workflows/validate-windows-ut.yml.
```

Comment:
Could we add the steps how to run the test manually in local so we do not need the pipeline to run unit tests to save time?